src/System.IO.FileSystem.AccessControl/src/System/IO/FileSystemAclExtensions.cs

```diff
@@ -56,7 +56,7 @@ public static FileSecurity GetAccessControl(this FileStream fileStream)
             {
                 throw new ObjectDisposedException(null, SR.ObjectDisposed_FileClosed);
             }
-            return new FileSecurity(handle, fileStream.Name, AccessControlSections.Access | AccessControlSections.Owner | AccessControlSections.Group);
```

Comment:
This is the only place where the constructor with the unused parameter was being called.